fitbenchmarking/fitbenchmarking
# Removing expert and mantid example scripts

#### Description of Work

Fixes #310 

Removes unused mantid and expert examples scripts

#### Testing Instructions

1. Check that the mention of either script is removed

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
